src/chart/model/baseline.model.ts

```diff
@@ -76,6 +78,9 @@ export class BaselineModel extends ChartBaseElement {
 		});
 	}
 
+	private dragStartCb = (chartPanComponent: ChartPanComponent) => chartPanComponent.deactivatePanHandlers();
```

Comment:
you can make `chartPanComponent` a class property, this way you wouldn't need to pass it via argument